components/BedPicker/BedPicker.vue

```diff
@@ -95,7 +97,11 @@ export default {
       bedList: [],
     };
   },
-  computed: {},
+  computed: {
+    isAllPicked() {
```

Comment:
I don't think we need this `computed` property.  The `prop` that it was bound to in `PickerBase` doesn't exist so I don't think this has any effect.  Please respond to this comment if I am missing something.  Otherwise, remove this `computed` property.